app/models/filter.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class Filter
```

Comment:
I think we can use Params object to handle this, instead of the PORO Filter.

Reply:
Updated this PR to use `keyword_params` instead of `Filter`.